ssl/ssl_transfer.cc

```diff
@@ -201,17 +203,16 @@ static int SSL3_STATE_to_bytes(SSL3_STATE *in, CBB *cbb) {
       !CBB_add_asn1_octet_string(&s3, in->previous_server_finished, PREV_FINISHED_MAX_SIZE) ||
       !CBB_add_asn1_uint64(&s3, in->previous_server_finished_len) ||
       !CBB_add_asn1_uint64(&s3, in->empty_record_count) ||
-      !CBB_add_asn1_uint64(&s3, in->warning_alert_count)) {
+      !CBB_add_asn1_uint64(&s3, in->warning_alert_count) ||
+      !CBB_add_asn1_uint64(&s3, in->total_renegotiations)) {
     OPENSSL_PUT_ERROR(SSL, ERR_R_MALLOC_FAILURE);
     return 0;
   }
 
-  if (in->established_session) {
```

Comment:
I don't have too much context from the previous PRs, but is this safe to remove?

Reply:
Yes, it's safe. `established_session` will always exist because ssl transfer only supports SSL after handshake completes(that means session is established). The previous check is not necessary.